src/Cli/BuildCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Dock\Cli;
+
+use Dock\Docker\Compose\Project;
+use Dock\Project\DockerComposeProjectManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class BuildCommand extends Command
+{
+    /**
+     * @var ProjectManager
+     */
+    private $projectManager;
+
+    /**
+     * @var Project
+     */
+    private $project;
+
+    /**
+     * @param DockerComposeProjectManager $projectManager
+     * @param Project        $project
+     */
+    public function __construct(DockerComposeProjectManager $projectManager, Project $project)
```

Comment:
I believe that you don't want to typehint `DockerComposeProjectManager` here, but your interface `ProjectBuildManager`. Then, you'll have to add this interface to the project managers used for tests to fix the tests :)